arches/app/src/components/ControlledListManager/ControlledListEditor.vue

```diff
@@ -0,0 +1,220 @@
+            :displayed-list="props.displayedList"
+            :editable="true"
+        />
+
+        <div
+            class="items"
+            style="margin: 1rem"
+        >
+            <h3 style="margin-top: 4rem; margin-left: 0">
+                {{ $gettext("List Item Editor") }}
+            </h3>
+            <Splitter
+                :pt="{
+                    gutter: { style: { background: lightGray } },
+                    gutterHandler: { style: { background: lightGray } },
+                }"
+            >
+                <SplitterPanel
+                    :size="30"
+                    :min-size="15"
+                >
+                    <Suspense>
+                        <SidepanelDataView
+                            :add-label="$gettext('Add New Item')"
+                            :delete-label="$gettext('Delete Item')"
+                            :delete-label-plural="$gettext('Delete Items')"
+                            :displayed-item="displayedItem"
+                            :create-item="createItem"
+                            :delete-items="deleteItems"
+                            :fetch-items="fetchItems"
+                            :item-label="$ngettext('item', 'items', items.length)"
+                            :items="items"
+                            :no-search-result-label="
+                                $gettext('No matching items')
+                            "
+                            :no-item-label="
+                                $gettext(
+                                    'Click &quot;Add New Item&quot; to start.'
+                                )
+                            "
+                            :set-displayed
+                        />
+                        <template #fallback>
+                            <SpinnerIcon />
+                        </template>
+                    </Suspense>
+                </SplitterPanel>
+
+                <SplitterPanel
+                    :size="75"
+                    :min-size="50"
+                    class="item-editor-container"
+                >
+                    <ItemEditor
+                        v-if="displayedItem"
+                        :item="displayedItem"
+                        :language-map="languageMap"
+                    />
+                    <ControlledListSplash
+                        v-else
+                        :description="
+                            $gettext('Select an item from the sidebar.')
+                        "
+                    />
+                </SplitterPanel>
+            </Splitter>
+        </div>
+        <Button
+            raised
+            class="button return"
+            @click="setEditing(false)"
+        >
+            {{ arches.translations.return }}
+        </Button>
+    </div>
+</template>
+
+<style scoped>
+h3 {
+    font-size: 1.5rem;
+}
+.p-splitter {
+    width: calc(100vw - 60px);
+    height: calc(100vh - 450px);
```

Comment:
if possible I'd like to avoid `calc` moving forward

Reply:
+1. Might want to pair on this with you at some point.

Reply:
I think for the most part, and if well thought out and done properly the use css grid or flex box, should allow us to avoid the use of calculated dimensions.  I usually see this when panels overlap when they shouldn't.

Reply:
In the next push, the only use of calc will be for the topmost plugin component, just to subtract the widths of the arches toolbars on top and left. Have we found any a better way of accounting for that?